### PR TITLE
[4.0] Rename label for menu type

### PR DESCRIPTION
**Pull request by chmst**
The label "menu type" is misleading. "type" is a general concept, here a unique name is required. It would be great if the description could be restored. At least the label should make clear that this name must be unique in the whole system. 

Test: code inspection

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_menus.ini [hunk 1]
```diff
@@ -125,7 +125,7 @@ COM_MENUS_MENU_CONFIRM_DELETE="Are you sure you want to delete these menus? Conf
 COM_MENUS_MENU_DETAILS="Menu Details"
 COM_MENUS_MENU_EXPORT_BUTTON="Download as Preset"
 COM_MENUS_MENU_ITEM_SAVE_SUCCESS="Menu item saved."
-COM_MENUS_MENU_MENUTYPE_LABEL="Menu Type"
+COM_MENUS_MENU_MENUTYPE_LABEL="Unique Menu Name"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_menus.ini]: or maybe "Unique Name"

Comment (chmst) [reply administrator/language/en-GB/en-GB.com_menus.ini]: Or so. For me the "unique" is important.

**Review by brianteeman — APPROVED** [review]

**Comment by brianteeman** [comment]
I never understood what this meant anyway

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 2b7ef8d99b3f10920235afffae90d60679a3aa58<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20726">issues.joomla.org/tracker/joomla-cms/20726</a>.</sub>

**Comment by chmst** [comment]
The whole handling is contrary of UX. This name is like a alias name - maybe better to use it like in content or elsewhere.

**Comment by brianteeman** [comment]
I honestly have no idea why we have it or what it is for. I am **guessing** it is an internal name - if so then there is no need to expose it at all and it can just be autogenerated like the alias

**Comment by Wang-Yu-Chao** [comment]
I have tested this item :white_check_mark: successfully on 2b7ef8d99b3f10920235afffae90d60679a3aa58<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20726">issues.joomla.org/tracker/joomla-cms/20726</a>.</sub>

The "Menu Type" label is replaced by "Unique Name" in the "Title" column.
image

**Comment by Quy** [comment]
The description in v3 refers it as `system name`. Would `System Name` be better?

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Comment by brianteeman** [comment]
@quy thi is an improvement as the old name didnt imply it must be unique

**Comment by Quy** [comment]
The search box has tooltip `Search in title and menu type.`  that needs to be updated.

**Comment by chmst** [comment]
I am not really happy with this solution. 
For users ist is important to know that this is a unique name and that it is used only by the system.
The term "menutype" is used many times especially in com_associations, so it has a meaning for developers. Why not add again the description for users? 
menutype-description

Then in https://github.com/joomla/joomla-cms/pull/20787#pullrequestreview-133457099 we could add the menutype like the alias in com_content/articles.

**Pull request merged**